<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mapwidget</class>
 <widget class="QWidget" name="Mapwidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWebEngineView" name="webEngineView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>991</width>
     <height>781</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>810</y>
     <width>790</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>起点：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="startInput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>终点：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="endInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="calculateButton">
      <property name="text">
       <string>计算距离时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>距离：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="distanceOutput"/>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="durationOutput"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
